### PR TITLE
Fix/Adjust attribute deletion feature for rejected relationships

### PR DESCRIPTION
**Pull request by britsta**

### CODE REVIEW

**Comment by jkoenig134** [comment]
@Magnus-Kuhn I fixed some things that were introduced by your new validations regarding canCreate of the Incoming- and OutgoingRequestController. Please review that properly and make sure I don't destroyed any ideas you had during implementing the initial validations.

File: packages/consumption/src/modules/requests/incoming/IncomingRequestsController.ts [hunk 1]
```diff
@@ -184,7 +184,12 @@ export class IncomingRequestsController extends ConsumptionBaseController {
 
         const relationship = await this.relationshipResolver.getRelationshipToIdentity(request.peer);
         // It is safe to decide an incoming Request when no Relationship is found as this is the case when the Request origins from onNewRelationship of the RelationshipTemplateContent
-        if (relationship && relationship.status !== RelationshipStatus.Active) {
+        const possibleStatuses =
+            request.source?.type === "RelationshipTemplate"
+                ? [RelationshipStatus.Active, RelationshipStatus.Rejected, RelationshipStatus.Revoked, RelationshipStatus.Terminated]
+                : [RelationshipStatus.Active];
+
```
Comment (Magnus-Kuhn) [inline packages/consumption/src/modules/requests/incoming/IncomingRequestsController.ts:191]: why should deciding relationship template requests still be allowed if the relationship is inactive?

Comment (jkoenig134) [reply packages/consumption/src/modules/requests/incoming/IncomingRequestsController.ts:191]: I think terminated could be wrong here. But rejected and revoked are like no relationship exists.

Comment (Magnus-Kuhn) [reply packages/consumption/src/modules/requests/incoming/IncomingRequestsController.ts:191]: Agreed that only Terminated is wrong. Is there a way (planned) to clean up rejected/revoked relationships?

Comment (jkoenig134) [reply packages/consumption/src/modules/requests/incoming/IncomingRequestsController.ts:191]: I mean, we could use your decompose logic for that. But please let US investigate that further before implementing sth like that.

Comment (jkoenig134) [reply packages/consumption/src/modules/requests/incoming/IncomingRequestsController.ts:189]: Alright, removed Terminated, should be fine now.

**Review by jkoenig134 — APPROVED** [review]
LGTM!

**Pull request merged**